frontend/packages/dev-console/src/components/svg/SvgBoxedText.tsx

```diff
@@ -14,6 +14,7 @@ export interface SvgBoxedTextProps {
   x?: number;
   y?: number;
   cornerRadius?: number;
+  icon?: any;
```

Comment:
```suggestion
  icon?: React.ReactNode;
```

Having a component accept children can potentially break the optimizations of `PureComponent` because child nodes are always new instances in the render function unless separated to be a constant.

Reply:
oops! this is not part of this PR. I was trying and pushed it accidently. :P